OpenRA.Game/Renderer.cs

```diff
@@ -91,6 +85,24 @@ static IGraphicsDevice CreateDevice(Assembly platformDll, int width, int height,
 			throw new InvalidOperationException("Renderer DLL is missing RendererAttribute to tell us what type to use!");
 		}
 
+		public readonly Size MinimumResolution = new Size(800, 600);
+
+		public readonly Size[] SuggestedResolutions = new Size[]
```

Comment:
These are all common screen resolutions, so it doesn't make sense to offer them.  There is no room for the window decorations!